test-network-k8s/scripts/chaincode.sh

```diff
@@ -70,120 +127,177 @@ function launch_chaincode_service() {
   pop_fn
 }
 
-function activate_chaincode_for() {
+# Copy the chaincode archive from the local host to the org admin
+function transfer_chaincode_archive_for() {
   local org=$1
+  local cc_archive=$2
+  local dirname=$(dirname $cc_archive)
+  local filename=$(basename $cc_archive)
+
+  push_fn "Transferring chaincode archive to ${org}"
+
+  # Like kubectl cp, but targeted to a deployment rather than an individual pod.
+  tar cf - -C ${dirname} ${filename} | kubectl -n $NS exec -i deploy/${org}-admin-cli -c main -- tar xvf -
```

Comment:
Yes, and no...   I am working on a new PR that will start using the `peer` binaries locally on the host OS, rather than copying ANYTHING to/from k8s with obscure and arcane tar commands.  I have NO idea what that `cp` line above is doing.  (It hurts that I wrote it... and still have no idea what it's doing...)